### PR TITLE
Bump versions to -rc.0

### PR DESCRIPTION
**Pull request by iclanton**

### CODE REVIEW

File: common/changes/@rushstack/heft-dev-cert-plugin/user-danade-FixWebpackStart_2022-11-14-20-23.json [hunk 1]
```diff
@@ -1,10 +0,0 @@
-{
```
Comment (D4N14L) [inline common/changes/@rushstack/heft-dev-cert-plugin/user-danade-FixWebpackStart_2022-11-14-20-23.json:1]: Expected?

File: heft-plugins/heft-api-extractor-plugin/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 # Change Log - @rushstack/heft-api-extractor-plugin
 
-This log was last generated on Fri, 13 Sep 2022 18:22:20 GMT and should not be manually modified.
+This log was last generated on Tue, 15 Nov 2022 22:59:35 GMT and should not be manually modified.
+
+## 0.1.0
```
Comment (D4N14L) [inline heft-plugins/heft-api-extractor-plugin/CHANGELOG.md:5]: Not RC?

**Comment by D4N14L** [comment]
You'll need to build the `install-test-workspace` project to update the lockfile it keeps, otherwise it'll warn and possibly break build on the pipeline.

File: common/changes/@rushstack/heft-dev-cert-plugin/user-danade-FixWebpackStart_2022-11-14-20-23.json (hunk 1 above)
Comment (iclanton) [reply common/changes/@rushstack/heft-dev-cert-plugin/user-danade-FixWebpackStart_2022-11-14-20-23.json:1]: Yeah

**Review by D4N14L — APPROVED** [review]

**Pull request merged**